apps/frontend/src/libs/components/input/input.tsx

```diff
@@ -60,15 +60,33 @@ const Input = <T extends FieldValues>({
 					</div>
 				)}
 
-				<input
-					autoComplete={autoComplete}
-					className={inputClassNames}
-					name={field.name}
-					onChange={field.onChange}
-					placeholder={placeholder}
-					type={type}
-					value={field.value}
-				/>
+				{isTextArea ? (
+					<textarea
+						className={getValidClassNames(
+							styles["input-field"],
+							styles["input-textarea"],
+						)}
+						name={field.name}
+						onChange={field.onChange}
+						placeholder={placeholder}
+						style={
+							{
+								"--rows": rows,
+							} as React.CSSProperties
+						}
```

Comment:
Can't we just pass rows to textarea?
```suggestion
rows={rows}
```